src/components/ManageUsersView.tsx

```diff
@@ -28,7 +28,7 @@ function ManageUsersView({ classes, removeUser, users }: ManageUsersProps) {
           color="primary"
           label={user}
           key={user}
-          onDelete={() => removeUser?.(user)}
+          onDelete={removeUser ? () => removeUser?.(user) : undefined}
```

Comment:
Why this change? I would expect `removeUser?.(user)` to handle the undefined check.

Reply:
This is for the additional fix I mentioned in the PR description. The "Chip" component is a material UI component and according to its documentation, it determines whether or not to display the "x" option based on whether or not the "onDelete" prop is defined.  So, even though removeUser was undefined in the DAC case, it was still returning a function  (something like`() => undefined`) and we instead want it to return just`undefined`. 

Before:
<img width="1077" alt="Screenshot 2024-07-29 at 1 44 03 PM" src="https://github.com/user-attachments/assets/26feb461-fd39-4c04-8a2b-b0d1591cf506">

With Change:
<img width="1071" alt="Screenshot 2024-07-29 at 1 42 43 PM" src="https://github.com/user-attachments/assets/98f2b336-1085-489c-83cc-b0d4983a0098">